WalletWasabi/Discoverability/CoordinatorAnnouncer.cs

```diff
@@ -0,0 +1,45 @@
+using System.Globalization;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using NBitcoin;
+using NNostr.Client;
+using NNostr.Client.Protocols;
+using WalletWasabi.Bases;
+using WalletWasabi.Logging;
+
+namespace WalletWasabi.Discoverability;
+
+public class CoordinatorAnnouncer(TimeSpan period, AnnouncerConfig config, Network network) : PeriodicRunner(period)
+{
+	protected override async Task ActionAsync(CancellationToken cancellationToken)
+	{
+		using var client = NostrClientFactory.Create(config.RelayUris.Select(x => new Uri(x)).ToArray());
+		var unsignedAnnouncementEvent = new NostrEvent
+		{
+			Kind = 15750,
+			Content = config.CoordinatorDescription,
+			Tags =
+			[
+				CreateTag("type", "wabisabi"),
+				CreateTag("network", network.ChainName.ToString().ToLower()),
+				CreateTag("endpoint", config.CoordinatorUri),
+				CreateTag("coordinationfee", config.CoordinationFee.ToString(CultureInfo.InvariantCulture)),
+				CreateTag("absolutemininputcount", config.AbsoluteMinInputCount.ToString(CultureInfo.InvariantCulture)),
+				CreateTag("readmore", config.ReadMoreUri)
```

Comment:
please add tag and config `Name`, it would be really useful for coordinator that use onion endpoints. Sorry I didn't think about it last time

Reply:
done